crates/cast/tests/cli/main.rs

```diff
@@ -1101,43 +1101,32 @@ casttest!(storage_layout, |_prj, cmd| {
     cmd.args([
             "storage",
             "--rpc-url",
-            next_rpc_endpoint(NamedChain::Optimism).as_str(),
+            next_rpc_endpoint(NamedChain::Mainnet).as_str(),
```

Comment:
I susoect this was just a flaky test and should've been passing if rerun failed job in CI, so not sure we should change the test. Wdyt?

Reply:
Sorry, yeah I didn't intend to commit it here, moved to https://github.com/foundry-rs/foundry/pull/9182